### DISCUSSION TITLE
.NET 4.6.1 support retires April 26,2022  Should Quartz target 4.6.2+ moving forward

### DISCUSSION THREAD

**Discussion by jafin** [quartznet/quartznet#1549] [Ideas] [answered]
Quartz at present targets `net461;net472;netstandard2.0`

From here https://docs.microsoft.com/en-us/lifecycle/faq/dotnet-framework

> NET Framework 4.5.2, 4.6, and 4.6.1 will retire on April 26, 2022. These specific releases were previously digitally signed using Secure Hash Algorithm (SHA-1) certificates. This algorithm is no longer deemed secure. Go [here](https://aka.ms/framework-452-46-461-eos-blog) to learn more about this retirement. Customers are encouraged to move to SHA-2. Go [here](https://docs.microsoft.com/en-us/azure/cloud-services/cloud-services-dotnet-install-dotnet) to learn about validating your application functionality on the newer runtime version.

Should quartz drop 4.6.1 and make 4.6.2 the minimum version?

4.6.2 support policy:

> .NET Framework 4.6.2: Support for .NET 4.6.2 follows the Lifecycle Policy of the parent OS. It is supported as a Windows component on the latest required operating system update for Windows Server 2008 SP2, Windows 7 SP1, Windows Server 2008 R2 SP1, Windows 8.1 Update, Windows Server 2012, Windows Server 2012 R2, Windows 10 (Version 1507), and Windows 10 (Version 1511). .NET 4.6.2 is also supported on Windows 10 Anniversary Update (Version 1607) and Windows Server 2016 operating systems.

**Comment by lahma** [comment, marked as answer]
Yes, for next 3.4 release we should have minimum of 4.6.2 and for v4 branch minimum probably should be 4.7.2 for now at least.
